cmake-tool/helpers/rootserver.cmake

```diff
@@ -30,7 +30,7 @@ mark_as_advanced(TLS_ROOTSERVER)
 find_file(UIMAGE_TOOL make-uimage PATHS "${CMAKE_CURRENT_LIST_DIR}" CMAKE_FIND_ROOT_PATH_BOTH)
 mark_as_advanced(UIMAGE_TOOL)
 include(CMakeDependentOption)
-cmake_dependent_option(UseRiscVOpenSBI "Use OpenSBI." ON DEPENDS "KernelArchRiscV" OFF)
+cmake_dependent_option(UseRiscVOpenSBI "Use OpenSBI." ON "KernelArchRiscV" OFF)
```

Comment:
This is the minimal fix. 
On the bigger picture, this is the only place where we use `cmake_dependent_option()`. Everywhere else we use our extended CMake function `config_option(). So maybe we should do it here also for consistency. Das might also explain where how wrong `DEPENDS` came use.

```
config_option(
    UseRiscVOpenSBI USE_RISCV_OPENSBI
    "Use OpenSBI."
    DEFAULT ON
    DEPENDS "KernelArchRiscV"
    DEFAULT_DISABLE OFF
)
```

Also, the better option name would be something like "PackageWithOpenSBI" actually.